tests/types/npm-test.json

```
@@ -209,7 +209,7 @@
       "description": "valid npm purl without version and with subpath, scope unencoded",
       "test_group": "advanced",
       "test_type": "roundtrip",
-      "input": "pkg:npm/@babel/core#/googleapis/api/annotations/",
+      "input": "pkg:npm/%40babel/core#/googleapis/api/annotations/",
```

This is an advanced test, so the input can be non-canonical, therefore we probably do not want to change that
```suggestion
      "input": "pkg:npm/@babel/core#/googleapis/api/annotations/",
```


The same comment applies to a few other inputs change proposals.